addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -1599,6 +1608,45 @@ public String convertElementToString(ZestElement element) {
                 : ZestJSON.toString(element);
     }
 
+    public void startClientRecording(ScriptNode scriptNode, String browserName, String uri) {
+        if (!isClientAccessible()) {
+            LOGGER.warn(ERROR_CLIENT);
```

Comment:
`    private static final String ERROR_CLIENT = "zest.dialog.script.error.client";`

Shouldn't this use i18n handling in one of the two locations?
I don't think you actually want to log the resource key name :wink:

Reply:
Fixed, we dont actually need this line so I've removed it.